e2e/cypress/integration/keyboard_shortcuts/backspace_spec.js

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @keyboard_shortcuts
+
+describe('Keyboard Shortcuts', () => {
+    let testUser;
+    let testTeam;
+    let publicChannel;
+
+    before(() => {
+        cy.apiInitSetup({loginAfter: true}).then(({team, channel, user}) => {
+            // # Visit a test channel
+            testTeam = team;
+            publicChannel = channel;
+            testUser = user;
+
+            cy.visit(`/${team.name}/channels/${channel.name}`);
+        });
+    });
+
+    it('Pressing the backspace key without an input focused should not send the browser back in history', () => {
+        // navigate to a couple of pages
+        cy.visit(`/${testTeam.name}/channels/town-square`);
+        cy.visit(`/${testTeam.name}/channels/off-topic`);
+        cy.visit(`/${testTeam.name}/channels/${publicChannel.name}`);
+        cy.visit(`/${testTeam.name}/channels/off-topic`);
+
+        cy.visit(`/${testTeam.name}/messages/@${testUser.username}`);
+        cy.url().should('include', `/${testTeam.name}/messages/@${testUser.username}`);
+
+        // click on the body to remove focus from the input key
+        cy.get('#post_textbox').clear().type('This is a normal sentence.').type('{backspace}{backspace}').blur();
```

Comment:
Please add a verification that the last 2 characters of the message have been removed.

Reply:
Also after the focus has been removed from post input, add another verification that no further truncation happened to the message in the input.